cms/static_src/sass/main.scss

```diff
@@ -2,3 +2,18 @@
 body {
     display: block;
 }
+
+// Custom CSS from Wagtail doc to enable responsive video embeds
+.responsive-object {
+    position: relative;
+}
+
+.responsive-object iframe,
+.responsive-object object,
+.responsive-object embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
```

Comment:
I suggest leaving this as is and using the DS component. https://service-manual.ons.gov.uk/design-system/components/video#example.example-video.nunjucks

I suggest raising a separate PR for it.

Reply:
I've added https://jira.ons.gov.uk/browse/CMS-204 for it